mmpose/datasets/datasets/top_down/topdown_crowdpose_dataset.py

```diff
@@ -45,9 +45,15 @@ def __init__(self,
                  img_prefix,
                  data_cfg,
                  pipeline,
-                 test_mode=False):
+                 test_mode=False,
+                 use_nms=True):
         super(TopDownCocoDataset, self).__init__(
-            ann_file, img_prefix, data_cfg, pipeline, test_mode=test_mode)
+            ann_file,
+            img_prefix,
+            data_cfg,
+            pipeline,
+            test_mode=test_mode,
+            use_nms=use_nms)
```

Comment:
Put use_nms under  dataset construction feels strange. How about making it an argument for evaluation?

Reply:
I'm thinking about putting 'use_nms' in **data_cfg** in config files of models, given that args like 'soft_nms' , 'nms_thr', 'oks_thr', 'vis_thr' and 'bbox_thr' are all specified in **data_cfg**. And since different evaluation methods are often unique to different datasets we used, it someway makes sense to put these args under dataset construction.